src/Graph.js

```diff
@@ -269,7 +273,6 @@ class DemandSupplyGraph extends Graph {
                 label: { position: 'rt', offset: [-10, -20] },
                 strokeColor: 'rgb(255, 127, 14)',
                 strokeWidth: 2,
-                snapToGrid: true,
```

Comment:
snapToGrid was causing the line's slope to get reset to a round value when the line is clicked. For example, a line of slope 1.5 would get rounded to 1. This appeared to be a jsxgraph quirk - i'm disabling this option for now since we don't really need it.